src/components/ministry/Testimonials.tsx

```diff
@@ -30,48 +37,43 @@ const Testimonials = () => {
                 blurDataURL={
```

Comment:
I noticed the image is not rendering properly... there should be a blur before the image loads